application/src/main/java/com/sanction/thunder/email/EmailConfiguration.java

```diff
@@ -28,4 +28,28 @@ public String getRegion() {
   public String getFromAddress() {
     return fromAddress;
   }
+
+  @NotEmpty
+  @JsonProperty("success-html")
+  private final String successHtmlPath = null;
+
+  public String getSuccessHtmlPath() {
+    return successHtmlPath;
+  }
+
+  @NotEmpty
```

Comment:
I have a little bit of a design rewrite that I would like you to consider. I think that we should allow for any files to be specified and then have the `resources` be the default. We don't have to enable files outside of resources just yet because I know there were some problems with that, but let's design with that in mind.

In that case, I would like you to make these config options optional (remove `@NotEmpty`). And then in the code, IF they are null we will load the `resources` ones that are the defaults. If they are not null, we try to load that file.

For now, we can expect that if a file is defined in the config, it will be in the resources folder so that we can avoid the problem of external files for now. But if we can go ahead and design it in this way right now, then it will be easy to make changes later to support files not in the resources folder.

What do you think @NickEckert?

Reply:
I like it